internal/commands/app/create.go

```diff
@@ -43,6 +49,7 @@ func (cmd *CommandCreate) Flags(fs *pflag.FlagSet) {
 	fs.VarP(&cmd.inputs.Environment, flagEnvironment, flagEnvironmentShort, flagEnvironmentUsage)
 	fs.StringVar(&cmd.inputs.Cluster, flagCluster, "", flagClusterUsage)
 	fs.StringVar(&cmd.inputs.DataLake, flagDataLake, "", flagDataLakeUsage)
+	fs.StringVar(&cmd.inputs.Template, flagTemplate, "", flagTemplateUsage)
```

Comment:
went with this single flag instead of the double `--template` and `--use-template` flags described in the ticket (see `resolveTemplateID` for usage) because this seemed simpler, but willing to discuss if there was a particular reason for that.